engines/saga/itedata.cpp

```diff
@@ -545,6 +586,11 @@ const char *sakkaStr[][NUM_SAKKA] = {
 		"Hey, non si dovrebbero aiutare i candidati!",
 		"Ragazzi! Questo dovrebbe essere un test!",
 		"Forza ragazzi, non si pu\225!"
+	},
+	{
+		"H\202, vous n'êtes pas suppos\202s aider les concurrents!",
```

Comment:
Shouldn't that be '\210tes' rather than 'êtes'?

Reply:
same